lib/SIL/Utils/PrunedLiveness.cpp

```diff
@@ -541,7 +541,8 @@ void MultiDefPrunedLiveness::findBoundariesInBlock(
     return;
   }
   // Handle a live-out or live-within block with potentially multiple defs
-  unsigned prevCount = boundary.deadDefs.size() + boundary.lastUsers.size();
+  SWIFT_ASSERT_ONLY(unsigned prevCount =
```

Comment:
This just suppresses an unused warning.  It could be written `(void)prevCount;` instead.